components/avatar.css

```diff
@@ -1,32 +1,37 @@
 .avatar-xsm {
+    box-sizing: content-box;
     border-radius: 50%;
     width: 2rem;
     height: 2rem;
     object-fit: cover;
 }
 
 .avatar-sm {
+    box-sizing: content-box;
     border-radius: 50%;
     width: 3rem;
     height: 3rem;
     object-fit: cover;
 }
 
 .avatar-md {
+    box-sizing: content-box;
     border-radius: 50%;
     width: 4rem;
     height: 4rem;
     object-fit: cover;
 }
 
 .avatar-lg {
+    box-sizing: content-box;
     border-radius: 50%;
```

Comment:
box-sizing, border-radius & object-fit are common properties. So you can put them in a single class to avoid repetition.

Reply:
okay